res/linux/org.mixxx.Mixxx.desktop

```diff
@@ -13,4 +13,5 @@ Terminal=false
 Icon=mixxx
 Type=Application
 StartupNotify=true
+StartupWMClass=org.mixxx.mixxx
```

Comment:
Why `org.mixxx.mixxx` instead of `org.mixxx.Mixxx` like the file name?

Did you check the actual WM Class of the window using xprop or Looking Glass? My system always shows `Mixxx` which differs from the proposed entry here.

Reply:
My bad, didn't notice the case. Yet works both ways.
Yes, i've checked, just commented on that.